presto-main/src/main/java/io/prestosql/execution/CreateViewTask.java

```diff
@@ -95,7 +96,7 @@ public String explain(CreateView statement, List<Expression> parameters)
             owner = Optional.empty();
         }
 
-        String data = codec.toJson(new ViewDefinition(sql, session.getCatalog(), session.getSchema(), columns, owner, !owner.isPresent()));
+        String data = codec.toJson(new ViewDefinition(sql, session.getCatalog().map(Name::getLegacyName), session.getSchema().map(Name::getLegacyName), columns, owner, !owner.isPresent()));
```

Comment:
We need to think through how to deal with backward compatibility for stored views, but we want to start storing the new delimited names going forward.